en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -293,6 +293,14 @@ page.
 > This demonstrates the concept of scope ([see scope below](#scope)). When inside a `<% loop %>` the scope of the template has changed to the
 > object that is being looped over.
 
+### Getting a count
+
+`$Count` is a special identifier in templates which allows you to get the number of items in any collection, even native PHP arrays.
+
+```ss
+$Children.Count
+```
```

Comment:
Leaving this not done for now pending a final decision on the main PR - but if it's merged as-is (i.e. wrapping in `ArrayList`) I'll make this change.
````suggestion
````